data/human/human missions.txt

```diff
@@ -4077,3 +4077,91 @@ mission "Laborer Payment"
 		conversation
 			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
 				decline
+
+
+
+mission "Block Hai Wormhole Warning"
+	landing
+	invisible
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		has "chosen sides"
+		not "main plot completed"
+	on offer
+		fail
+
+mission "Hai Wormhole Warning"
+	minor
+	source
+		government "Republic"
+		not attributes "deep"
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		not "Block Hai Wormhole Warning: offered"
+		not "Wanderers: Alpha Surveillance E: offered"
+	on offer
+		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
+		conversation
+			`Upon entering the spaceport, you're stopped by Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
```

Comment:
```suggestion
			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>, I'd like to speak with you for a few minutes... In private." He motions toward a Frigate on a landing pad behind you.`
```

Reply:
Edited your suggestion: I'd like to keep the dialog as three separate sentences.